javascriptlib/src/mill/javascriptlib/TypeScriptModule.scala

```diff
@@ -661,7 +709,7 @@ trait TypeScriptModule extends Module { outer =>
 
       val combinedCompilerOptions: Map[String, ujson.Value] =
         outer.compilerOptions() ++ compilerOptions() ++ Map(
-          "declarationDir" -> outer.declarationDir(),
+          "declarationDir" -> ujson.Str(outer.declarationDir()),
```

Comment:
revert

Reply:
declarationDir was changed from `T[usjon.Value]` to `T[String]`

Reply:
ujson does not see to correctly infer types, it will throw the error.

```text
Found: Map[String, String | ujson.Obj]
Required: IterableOnce[(String, ujson.Value)]
```